pint_server/database.py

```diff
@@ -39,21 +39,59 @@
         VersionsModel
     )
 
-
 def get_environ_or_bust(key_name):
     assert key_name in os.environ, 'Environment variable %s is required.' % (
         key_name)
     return os.environ.get(key_name)
 
 
-def create_postgres_url_from_env():
-    db_user = get_environ_or_bust('POSTGRES_USER')
-    db_password = get_environ_or_bust('POSTGRES_PASSWORD')
-    db_name = get_environ_or_bust('POSTGRES_DB')
-    db_host = get_environ_or_bust('POSTGRES_HOST')
-    db_port = os.environ.get('POSTGRES_PORT', 5432)
-    db_ssl_mode = os.environ.get('POSTGRES_SSL_MODE', None)
-    db_root_cert = os.environ.get('POSTGRES_SSL_ROOT_CERTIFICATE', None)
+def create_db_logger(outputfile):
+    """Function to setup logging of SQL statements
+
+    Args:
+        outputfile (filepath): The filepath where the SQL statements will be
+            logged
+    """
+    if not outputfile:
+        return
+
+    db_log_file_name = outputfile
+    db_handler_log_level = logging.INFO
+    db_logger_log_level = logging.DEBUG
+
+    db_handler = logging.FileHandler(db_log_file_name)
+    db_handler.setLevel(db_handler_log_level)
+
+    db_logger = logging.getLogger('sqlalchemy')
```

Comment:
'sqlalchemy.engine' right?

Reply:
@KeithMnemonic would have to answer the specifics of that one, but my understanding was that the selections were made to ensure that we capture the specific requested data as log entries.